cloudinit/config/cc_apt_configure.py

```diff
@@ -55,6 +55,7 @@
             ``Additional apt configuration`` example."""
     ),
     "distros": distros,
+    "activate_by_schema_keys": ["apt"],
```

Comment:
We can't define apt here as the cc_apt_configure module does perform operations to setup mirrors and or proxies on #cloud-config without 'apt' keys due to [sould_configure_on_empty_apt](https://github.com/canonical/cloud-init/blob/main/cloudinit/config/cc_apt_configure.py#L192)
```
2022-07-20 15:32:56,936 - cc_apt_configure.py[DEBUG]: Apt Mirror info: {'primary': 'http://archive.ubuntu.com/ubuntu', 'security': 'http://security.ubuntu.com/ubuntu', 'PRIMARY': 'http://archive.ubuntu.com/ubuntu', 'SECURITY': 'http://security.ubuntu.com/ubuntu', 'MIRROR': 'http://archive.ubuntu.com/ubuntu'}
2022-07-20 15:32:56,936 - cc_apt_configure.py[INFO]: No custom template provided, fall back to builtin
2022-07-20 15:32:56,937 - util.py[DEBUG]: Reading from /etc/cloud/templates/sources.list.ubuntu.tmpl (quiet=False)
2022-07-20 15:32:56,937 - util.py[DEBUG]: Read 2504 bytes from /etc/cloud/templates/sources.list.ubuntu.tmpl
2022-07-20 15:32:56,959 - util.py[DEBUG]: Writing to /etc/apt/sources.list - wb: [644] 2820 bytes
```

Reply:
done